packages/suite/src/actions/wallet/send/sendFormSolanaActions.ts

```diff
@@ -1,13 +1,145 @@
+import TrezorConnect, { FeeLevel, TokenInfo } from '@trezor/connect';
 import { Transaction, SystemProgram, PublicKey } from '@solana/web3.js';
 import {
     FormState,
     PrecomposedTransactionFinal,
+    ComposeActionContext,
+    ExternalOutput,
+    PrecomposedTransaction,
+    PrecomposedLevels,
 } from '@suite-common/wallet-types';
 import { notificationsActions } from '@suite-common/toast-notifications';
 import { Dispatch, GetState } from 'src/types/suite';
 import {
+    calculateMax,
+    calculateSolFee,
+    calculateTotal,
+    formatAmount,
+    getExternalComposeOutput,
     getLamportsFromSol,
 } from '@suite-common/wallet-utils';
+import BigNumber from 'bignumber.js';
+
+// TODO(vl): Implement send-max
```

Comment:
How hard would it be to do the send max in this PR (next commit)? I am asking because if we wont do it now, I would probably be for removing the `max` related logic down below. Honestly I would be for removing that logic either way and then reintroducing it in the next commit together with a legit support for send-max. Wha do you think?

Reply:
Send max actually works out of the box. The todo comment seems to have slipped through when I was rebasing.